src/billing/check.ts

```diff
@@ -0,0 +1,36 @@
+import {Context} from '../context';
+import {SessionInterface} from '../auth/session/types';
+
+import {hasActivePayment} from './has_active_payment';
+import {requestPayment} from './request_payment';
+
+interface CheckInterface {
+  session: SessionInterface;
+  isTest: boolean;
+}
+
+interface CheckReturn {
+  hasPayment: boolean;
+  confirmationUrl?: string;
+}
+
+export async function check({
+  session,
+  isTest,
```

Comment:
Should we default `isTest` to `true` to avoid any potential mishaps with testing during development?